packages/smooth_app/lib/helpers/product_cards_helper.dart

```diff
@@ -6,9 +6,24 @@ import 'package:smooth_app/data_models/product_image_data.dart';
 import 'package:smooth_app/generic_lib/design_constants.dart';
 import 'package:smooth_app/generic_lib/widgets/smooth_card.dart';
 
+RegExp _commaRegex = RegExp(r'\s*,\s*');
+
 String getProductName(Product product, AppLocalizations appLocalizations) =>
     product.productName ?? appLocalizations.unknownProductName;
 
+String getProductBrands(Product product, AppLocalizations appLocalizations) {
+  final String? brands = product.brands;
+  if (brands == null) {
+    return appLocalizations.unknownBrand;
+  } else {
+    return formatProductBrands(brands);
+  }
+}
+
+String formatProductBrands(String brands) =>
+    // Correctly format commas between words
+    brands.replaceAll(_commaRegex, ', ');
```

Comment:
This should be done via translations.
We cannot say that all languages have no space before the comma and always a space after

Reply:
So `', '` should be added to `lib/l10n/app_en.arb`